commit 0e3ae6f8f4a4d2a4419e43c09b1e3169e44c7389
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Tue Jan 16 22:44:57 2018 +0100

    Fix firmlinks showing up in /proc/mounts
    
    mtab checks for fsys_get_options.
    
    * trans/firmlink.c: Include <argz.h>.
    (trivfs_append_args): New function.

diff --git a/trans/firmlink.c b/trans/firmlink.c
index ca094ecf..1c656a6f 100644
--- a/trans/firmlink.c
+++ b/trans/firmlink.c
@@ -25,6 +25,7 @@
 #include <string.h>
 #include <fcntl.h>
 #include <argp.h>
+#include <argz.h>
 #include <error.h>
 #include <sys/mman.h>
 
@@ -287,6 +288,13 @@ trivfs_S_io_select_timeout (struct trivfs_protid *cred,
   return trivfs_S_io_select (cred, reply, reply_type, type);
 }
 
+error_t
+trivfs_append_args (struct trivfs_control *fsys,
+		    char **argz, size_t *argz_len)
+{
+  return argz_add (argz, argz_len, "");
+}
+
 error_t trivfs_get_source (char *source, size_t source_len)
 {
   strncpy (source, target, source_len - 1);
